site-src/concepts/guidelines.md

```diff
@@ -118,6 +140,18 @@ The "implementation-specific" designation allows a CORE or EXTENDED feature to
 be well-defined taking into account the realities of some features that are
 mostly but not entirely portable.
 
+## Limitations of Webhook and CRD Validation
+CRD and webhook validation is not the final validation i.e. webhook is "nice UX"
```

Comment:
We should clarify why we are making this statement, otherwise the reader will be somewhat left with a directive without the associated reasoning.

- Are we saying that webhooks are inherently not reliable because people can uninstall them?
- Or is it because of API revisions,

Or both...

Reply:
Reworked a lot of this section, hopefully it's clearer now.